src/main/java/uk/ac/cam/cl/dtg/isaac/api/EventsFacade.java

```diff
@@ -214,15 +214,15 @@ public final Response getEvents(@Context final HttpServletRequest request, @Quer
         fieldsToMatch.put(TYPE_FIELDNAME, Arrays.asList(EVENT_TYPE));
 
         Map<String, AbstractFilterInstruction> filterInstructions = null;
-        if (null == showActiveOnly || showActiveOnly) {
+        if (null != showActiveOnly && showActiveOnly) {
```

Comment:
Perhaps the correct fix is to alter the value of `showActiveOnly` to be `showActiveOnly = showActiveOnly || null == showInactiveOnly`, so that it is the default if the other one is not provided.

Reply:
Agreed. Can we also have it as `showActiveOnly = showActiveOnly || (null == showInactiveOnly)` please? I know Java does not support null coalescing, but it took my eyes crossing a little too much to understand that expression correctly.

Reply:
Having played with this some more, I see why Connor did what he did. My suggestion doesn't work because (a) the variables is `final` and (b) it leads to NullPointerExceptions.
So I'm just going with what he did, since the frontend always specifies both.